pkgs/by-name/in/invoiceplane/node_switch_to_sass.patch

```diff
@@ -0,0 +1,3022 @@
+diff --git a/Gruntfile.js b/Gruntfile.js
+index a201bafe..f11bf5bf 100644
+--- a/Gruntfile.js
++++ b/Gruntfile.js
+@@ -1,6 +1,6 @@
+ "use strict";
+ module.exports = function(grunt) {
+-  const sass = require("node-sass");
++  const sass = require("sass");
+ 
+   // Load grunt tasks automatically
+   require("load-grunt-tasks")(grunt);
+diff --git a/package.json b/package.json
+index 878e4233..1a16507b 100644
+--- a/package.json
++++ b/package.json
+@@ -31,8 +31,8 @@
+         "jquery-ui": "1.14.1",
+         "js-cookie": "2.2",
+         "load-grunt-tasks": "5.1",
+-        "node-sass": "9.0",
+         "postcss": "8.4",
++        "sass": "^1.89.2",
+         "select2": "4.1.0-rc.0",
+         "zxcvbn": "4.4"
+     },
```

Comment:
Can we see if upstream reacts about this change in a few days? Maybe open a PR? it will likely very quickly collect merge conflicts but just merging this as is into nixpkgs feels kinda wrong :/

Reply:
Created an upstream pull-request for it https://github.com/InvoicePlane/InvoicePlane/pull/1277